src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -172,8 +173,8 @@ private void runGcloudCommand(List<String> args, String... topLevelCommand)
   // class's main configured ProcessRunner should be used.
   private String runSynchronousGcloudCommand(List<String> args)
       throws ProcessRunnerException {
-    validateCloudSdk();
-
+    validateCloudSdkLocation();
```

Comment:
We should probably not have used unchecked exceptions here in the first place, but c'est la code. I made the version exception a subclass of the not found exception so clients won't get immediately blown up.